CONTRIBUTING.md

```diff
@@ -29,24 +29,28 @@ The following text equips you with knowledge that makes contributing to ilastik
    gh repo clone ilastik
    ```
 
-1. Configure conda to use [strict channel priority](https://conda.io/projects/conda/en/latest/user-guide/tasks/manage-channels.html#strict-channel-priority):
+1. We recommend to downloading/using Mambaforge to interact with conda.
+   Mambaforge already comes configured correctly (this step can be skipped in this case).
+   If you are using [Miniconda][miniconda] instead, please make sure to configure conda to use [strict channel priority][strict-channel-priority], and install the fast [Mamba][mamba] solver for conda environments:
 
    ```
    conda config --set channel_priority strict
+   conda install --name-base -c conda-forge mamba
    ```
 
 1. Install [conda-build][conda-build] in order to access the [conda develop][conda-develop] command:
 
    ```
-   conda install --name base -c conda-forge conda-build
+   mamba install --name base -c conda-forge conda-build
```

Comment:
Are we also moving to `boa` from `conda-build`?

Reply:
when using conda-develop it doesn't make a difference. but yeah, for actual building packages I'm using boa right now.